packages/ceramic-common/src/ceramic-api.ts

```diff
@@ -23,25 +24,17 @@ export interface PinApi {
     ls(docId?: string): Promise<AsyncIterable<string>>;
 }
 
-// TODO handle errors as well
-export interface JsonRpc2Response {
-    'id': string;
-    'json-rpc': string;
-    'result': object;
-}
-
 /**
  * Describes DID provider instance
  */
-export interface DIDProvider {
-    send(jsonReq: object): JsonRpc2Response;
-}
+export type { DIDProvider } from 'dids'
 
 /**
  * Describes Ceramic node API
  */
 export interface CeramicApi {
     pin: PinApi;
+    user?: DID;
```

Comment:
do we need `user` here?

Reply:
It's used in various places in IDX to check if the user is authenticated, access its `DID` to set the owner when creating documents, resolve the DID document, etc.

Reply:
yep, this is just a reference to the js-did instance.

Reply:
makes sense! cool!